webmessaging/Channel_postMessage_Blob.htm

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title> postMessage() with a Blob </title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id=log></div>
+<script>
+    var TARGET = null;
+    var SOURCE = null;
+    var description = "Messages can contain Blobs.";
+
+    var t = async_test("Test Description: " + description);
+
+    var channel = new MessageChannel();
+    SOURCE = channel.port1;
+    TARGET = channel.port2;
+    TARGET.start();
+    TARGET.addEventListener("message", t.step_func(TestMessageEvent), true);
+
+    (function() {
+      SOURCE.postMessage({blob: new Blob(['foo', 'bar'])});
+    })();
+    // TODO(Issue #7899): Change to some sort of cross-browser GC trigger.
```

Comment:
Can you make this a URL? Reaching the issue from this line took longer than I would have liked, and these things can become ambiguous over time as repos merge or move.

Reply:
Done (once Blink WPT Bot syncs the CL to this PR again)